corehq/apps/es/users.py

```diff
@@ -109,7 +109,8 @@ def user_ids(user_ids):
     return filters.term("_id", list(user_ids))
 
 
-def location(location_id):
+def primary_location(location_id):
+    # by primary location
```

Comment:
actually I imagine most things currently using this filter would prefer to return users where any of their assigned locations match the location(s) passed to the filter.

That is, any users where `user.location_ids` intersects at all with `location_ids`